consensus/replay.go

```diff
@@ -492,15 +496,19 @@ func (h *Handshaker) replayBlocks(
 
 // ApplyBlock on the proxyApp with the last block.
 func (h *Handshaker) replayBlock(state sm.State, height int64, proxyApp proxy.AppConnConsensus) (sm.State, error) {
-	block := h.store.LoadBlock(height)
+	block, err := h.store.LoadBlock(context.TODO(), height)
+	if err != nil {
+		// todo(evan): handle/bubble this error
+		panic(err)
+		// return sm.State{}, err
+	}
```

Comment:
Why panic and not return?

Reply:
You can return an error in this func

Reply:
I was initially tepid about returning errors, as now that the blockstore is offline if the block cannot be saved or loaded then something is likely seriously wrong. I definitely should have updated my todos before pushing, thanks for catching this, and it is resolved here https://github.com/celestiaorg/lazyledger-core/pull/374/commits/21a18b243857f1d80ce8f47067a4b863cf2c2ca4